packages/@aws-cdk/aws-iotevents/test/input.test.ts

```diff
@@ -39,9 +42,41 @@ test('can get input name', () => {
   });
 });
 
-test('can set physical name', () => {
-  const stack = new cdk.Stack();
+test('can get input ARN', () => {
+  // GIVEN
+  const input = new iotevents.Input(stack, 'MyInput', {
+    attributeJsonPaths: ['payload.temperature'],
+  });
 
+  // WHEN
+  new cdk.CfnResource(stack, 'Res', {
+    type: 'Test::Resource',
+    properties: {
+      InputArn: input.inputArn,
+    },
+  });
+
+  // THEN
+  Template.fromStack(stack).hasResourceProperties('Test::Resource', {
+    InputArn: {
+      'Fn::Join': [
+        '',
+        [
+          'arn:',
+          { Ref: 'AWS::Partition' },
+          ':iotevents:',
+          { Ref: 'AWS::Region' },
+          ':',
+          { Ref: 'AWS::AccountId' },
+          ':input/',
+          { Ref: 'MyInput08947B23' },
+        ],
+      ],
```

Comment:
Let's conserve vertical space where it doesn't hurt readability:

```suggestion
      'Fn::Join': ['', [
        'arn:',
        { Ref: 'AWS::Partition' },
        ':iotevents:',
        { Ref: 'AWS::Region' },
        ':',
        { Ref: 'AWS::AccountId' },
        ':input/',
        { Ref: 'MyInput08947B23' },
      ]],
```